contracts/test/InterchainTokenServiceTest.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { InterchainTokenService } from '../InterchainTokenService.sol';
+
+contract InterchainTokenServiceTest is InterchainTokenService {
```

Comment:
we've been using Test prefix for test contracts so far. use that for consistency?
```suggestion
contract TestInterchainTokenService is InterchainTokenService {
```